src/Misc/layoutbin/daemon.svc.sh.template

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
```

Comment:
Why not more portable `/usr/bin/env bash` shebang?

Reply:
I originally copied it from _systemd.svc.sh.template_, which also uses `#!/bin/bash`, but your portable solution seems more appropriate for FreeBSD.